src/CSnakes.Runtime.Tests/Python/ImportTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+using CSnakes.Runtime.Python;
+
+namespace CSnakes.Runtime.Tests.Python;
+public class ImportTests : RuntimeTestBase
+{
+    [Fact]
+    public void TestImportModule()
+    {
+        using (GIL.Acquire())
+        {
+            using PyObject? sys = Import.ImportModule("sys");
```

Comment:
The `?` is redundant since `Import.ImportModule` returns a non-nullable `PyObject`:

```suggestion
            using PyObject sys = Import.ImportModule("sys");
```